src/node_zlib.cc

```diff
@@ -304,10 +314,10 @@ class ZCtx : public AsyncWrap {
 
     HandleScope handle_scope(env->isolate());
     Context::Scope context_scope(env->context());
-
     if (!CheckError(ctx))
       return;
 
+
```

Comment:
Can you undo the whitespace changes?  It makes the diff noisier than it needs to be.